client/src/pages/Job/index.tsx

```diff
@@ -24,68 +27,108 @@ export default function Job() {
     let h2Size: string = mobile ? ".75rem": "2rem";
     let imgSize: number = mobile ? 45 : 75;
```

Comment:
Would it be better to add this to the styles or themes files so that whenever someone else needs responsive CSS they don't need to rewrite or define their own (ie inconsistencies)

Reply:
I get what you're saying, but these responsive values I picked are kinda just what work for this specifically. They would not work generically.